modules/navigation.js

```diff
@@ -0,0 +1,29 @@
+export default function loadNavigation() {
```

Comment:
- [ ] One of the objectives for this project is to use ES6 syntax, so kindly use an arrow function instead of the function keyword
![image](https://user-images.githubusercontent.com/73140461/148828227-11ff80b2-2980-423d-b473-0b6ff6899504.png)